integrations/openai/src/databricks_openai/mcp_server_tookit.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable, List
+
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.credentials_provider import OAuthCredentialsProvider
+from databricks_mcp import DatabricksMCPClient
+from openai.types.chat import ChatCompletionToolParam
+from pydantic import BaseModel
+
+
+class ToolInfo(BaseModel):
```

Comment:
Should we just follow the pattern + APIs exposed by `databricks_openai.VectorSearchRetreiverTool`?